packages/web/src/components/notification/Notification/components/icons.module.css

```diff
@@ -67,6 +67,15 @@
   margin-right: var(--unit-2);
 }
 
+.iconCart {
+  composes: icon;
+  height: 24px;
```

Comment:
I think we're moving towards setting sizes in grid units (`var(--unit-6)`), and isn't the icon square? So inheriting the width of 30px is going to stretch this icon horizontally/

Reply:
going to just leave this as composes:icon, i think we were just trying to follow what other icons were doing but looks fine just inheriting icon ?
<img width="417" alt="Screenshot 2023-07-22 at 9 50 52 PM" src="https://github.com/AudiusProject/audius-client/assets/109105561/efbb6834-d4c4-4b4f-a02e-03ae5049f217">